### PR TITLE
try to add the Rust/coreutils

### PR DESCRIPTION
**Pull request by sylvestre**

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
Can you change the headers to include the standard google copyright statement.

File: projects/rust-coreutils/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 Sylvestre Ledru <sylvestre@debian.org>
```
Comment (jonathanmetzman) [inline projects/rust-coreutils/Dockerfile:1]: It has to be `# Copyright 2023 Google LLC` for us to accept this.
If youre uncomfortable with this, you can put the real content of build.sh in a script in the original repo and use build.sh here to call into that script.

Comment (sylvestre) [reply projects/rust-coreutils/Dockerfile:1]: oh ok :) fun
no worries, I don't care about this :)

Comment (sylvestre) [reply projects/rust-coreutils/Dockerfile]: fixed 
btw, the bot could give more helpful error messages :)

**Comment by sylvestre** [comment]
@jonathanmetzman I think we are good now :)

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Comment by sylvestre** [comment]
@jonathanmetzman quick question about accessing the results
I have a professional account on oss fuzz ( sledru@mozilla.com ) but as this is a personal project, I am wondering what would be the best approach to see the rust-coreutils results?
creating a sylvestre@debian.org account on clusterfuzz?

**Comment by jonathanmetzman** [comment]
Yeah probably a good idea to use a different email account that is associated with github account or is google apps or gmail.

**Pull request merged**